group_vars/tcbsd_vms/vars.yml

```diff
@@ -10,6 +10,17 @@ ansible_python_interpreter: /usr/local/bin/python3
 # point.
 enable_freebsd_packages: true
 
+# psproxy and psntp are currently needed to get bsd and package updates while on the lcls cds networks
+use_psproxy: false
+use_psntp: false
+
+# set static IP on x000 (mac id 2)
+x000_static:
```

Comment:
I suppose this would need to be filled, but it doesn't break anything now because we don't use the VMs?

Reply:
The intent here is that it is set to empty string, but maybe this is fundamentally unclear and I should switch these to booleans

Reply:
I'll do this clarity change and then I think this PR is done after that